2019-01-08 06:42:27,140 [INFO] from application in play-dev-mode-akka.actor.default-dispatcher-8 - Creating Pool for datasource 'default'
2019-01-08 06:42:27,329 [INFO] from play.api.db.DefaultDBApi in play-dev-mode-akka.actor.default-dispatcher-8 - Database [default] connected at jdbc:postgresql://localhost/ceng472
2019-01-08 06:42:28,182 [ERROR] from org.jdbcdslog.StatementLogger in play-dev-mode-akka.actor.default-dispatcher-8 - java.sql.Statement.executeQuery: select id, hash, apply_script, revert_script, state, last_problem from play_evolutions where state like 'applying_%';
throws exception: org.postgresql.util.PSQLException: ERROR: relation "play_evolutions" does not exist
  Position: 72
org.postgresql.util.PSQLException: ERROR: relation "play_evolutions" does not exist
  Position: 72
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2422)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2167)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:306)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:441)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:365)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:307)
	at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:293)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:270)
	at org.postgresql.jdbc.PgStatement.executeQuery(PgStatement.java:224)
	at com.zaxxer.hikari.pool.ProxyStatement.executeQuery(ProxyStatement.java:111)
	at com.zaxxer.hikari.pool.HikariProxyStatement.executeQuery(HikariProxyStatement.java)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.jdbcdslog.StatementLoggingHandler.invoke(StatementLoggingHandler.java:27)
	at com.sun.proxy.$Proxy39.executeQuery(Unknown Source)
	at play.api.db.evolutions.DatabaseEvolutions.executeQuery(EvolutionsApi.scala:316)
	at play.api.db.evolutions.DatabaseEvolutions.checkEvolutionsState(EvolutionsApi.scala:270)
	at play.api.db.evolutions.DatabaseEvolutions.databaseEvolutions(EvolutionsApi.scala:135)
	at play.api.db.evolutions.DatabaseEvolutions.scripts(EvolutionsApi.scala:110)
	at play.api.db.evolutions.DatabaseEvolutions.scripts(EvolutionsApi.scala:125)
	at play.api.db.evolutions.DefaultEvolutionsApi.scripts(EvolutionsApi.scala:90)
	at play.api.db.evolutions.ApplicationEvolutions.$anonfun$runEvolutions$1(ApplicationEvolutions.scala:53)
	at play.api.db.evolutions.ApplicationEvolutions.withLock(ApplicationEvolutions.scala:100)
	at play.api.db.evolutions.ApplicationEvolutions.runEvolutions(ApplicationEvolutions.scala:49)
	at play.api.db.evolutions.ApplicationEvolutions.$anonfun$start$1(ApplicationEvolutions.scala:42)
	at play.api.db.evolutions.ApplicationEvolutions.$anonfun$start$1$adapted(ApplicationEvolutions.scala:42)
	at scala.collection.immutable.List.foreach(List.scala:389)
	at play.api.db.evolutions.ApplicationEvolutions.start(ApplicationEvolutions.scala:42)
	at play.api.db.evolutions.ApplicationEvolutions.<init>(ApplicationEvolutions.scala:151)
	at play.api.db.evolutions.ApplicationEvolutionsProvider.get$lzycompute(EvolutionsModule.scala:50)
	at play.api.db.evolutions.ApplicationEvolutionsProvider.get(EvolutionsModule.scala:49)
	at play.api.db.evolutions.ApplicationEvolutionsProvider.get(EvolutionsModule.scala:40)
	at com.google.inject.internal.ProviderInternalFactory.provision(ProviderInternalFactory.java:81)
	at com.google.inject.internal.BoundProviderFactory.provision(BoundProviderFactory.java:72)
	at com.google.inject.internal.ProviderInternalFactory.circularGet(ProviderInternalFactory.java:61)
	at com.google.inject.internal.BoundProviderFactory.get(BoundProviderFactory.java:62)
	at com.google.inject.internal.ProviderToInternalFactoryAdapter$1.call(ProviderToInternalFactoryAdapter.java:46)
	at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1092)
	at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40)
	at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:194)
	at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:41)
	at com.google.inject.internal.InternalInjectorCreator$1.call(InternalInjectorCreator.java:205)
	at com.google.inject.internal.InternalInjectorCreator$1.call(InternalInjectorCreator.java:199)
	at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1085)
	at com.google.inject.internal.InternalInjectorCreator.loadEagerSingletons(InternalInjectorCreator.java:199)
	at com.google.inject.internal.InternalInjectorCreator.injectDynamically(InternalInjectorCreator.java:180)
	at com.google.inject.internal.InternalInjectorCreator.build(InternalInjectorCreator.java:110)
	at com.google.inject.Guice.createInjector(Guice.java:99)
	at com.google.inject.Guice.createInjector(Guice.java:84)
	at play.api.inject.guice.GuiceBuilder.injector(GuiceInjectorBuilder.scala:185)
	at play.api.inject.guice.GuiceApplicationBuilder.build(GuiceApplicationBuilder.scala:137)
	at play.api.inject.guice.GuiceApplicationLoader.load(GuiceApplicationLoader.scala:21)
	at play.core.server.DevServerStart$$anon$1.$anonfun$reload$3(DevServerStart.scala:174)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.core.server.DevServerStart$$anon$1.reload(DevServerStart.scala:171)
	at play.core.server.DevServerStart$$anon$1.get(DevServerStart.scala:124)
	at play.core.server.AkkaHttpServer.handleRequest(AkkaHttpServer.scala:202)
	at play.core.server.AkkaHttpServer.$anonfun$createServerBinding$1(AkkaHttpServer.scala:117)
	at akka.stream.impl.fusing.MapAsync$$anon$25.onPush(Ops.scala:1194)
	at akka.stream.impl.fusing.GraphInterpreter.processPush(GraphInterpreter.scala:519)
	at akka.stream.impl.fusing.GraphInterpreter.processEvent(GraphInterpreter.scala:482)
	at akka.stream.impl.fusing.GraphInterpreter.execute(GraphInterpreter.scala:378)
	at akka.stream.impl.fusing.GraphInterpreterShell.runBatch(ActorGraphInterpreter.scala:585)
	at akka.stream.impl.fusing.GraphInterpreterShell$AsyncInput.execute(ActorGraphInterpreter.scala:469)
	at akka.stream.impl.fusing.GraphInterpreterShell.processEvent(ActorGraphInterpreter.scala:560)
	at akka.stream.impl.fusing.ActorGraphInterpreter.akka$stream$impl$fusing$ActorGraphInterpreter$$processEvent(ActorGraphInterpreter.scala:742)
	at akka.stream.impl.fusing.ActorGraphInterpreter$$anonfun$receive$1.applyOrElse(ActorGraphInterpreter.scala:757)
	at akka.actor.Actor.aroundReceive(Actor.scala:517)
	at akka.actor.Actor.aroundReceive$(Actor.scala:515)
	at akka.stream.impl.fusing.ActorGraphInterpreter.aroundReceive(ActorGraphInterpreter.scala:667)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:590)
	at akka.actor.ActorCell.invoke(ActorCell.scala:559)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:257)
	at akka.dispatch.Mailbox.run(Mailbox.scala:224)
	at akka.dispatch.Mailbox.exec(Mailbox.scala:234)
	at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
2019-01-08 06:42:28,520 [INFO] from application in play-dev-mode-akka.actor.default-dispatcher-8 - DB handler is started...
2019-01-08 06:42:28,576 [INFO] from application in play-dev-mode-akka.actor.default-dispatcher-8 - Test doctors and patients are missing. Inserting default values...
2019-01-08 06:42:28,576 [DEBUG] from application in play-dev-mode-akka.actor.default-dispatcher-8 - Generating doctor user.
2019-01-08 06:42:28,663 [DEBUG] from application in play-dev-mode-akka.actor.default-dispatcher-8 - Generating doctor user.
2019-01-08 06:42:28,670 [DEBUG] from application in play-dev-mode-akka.actor.default-dispatcher-8 - Generating doctor user.
2019-01-08 06:42:28,676 [DEBUG] from application in play-dev-mode-akka.actor.default-dispatcher-8 - Generating patient user.
2019-01-08 06:42:28,711 [DEBUG] from application in play-dev-mode-akka.actor.default-dispatcher-8 - Generating patient user.
2019-01-08 06:42:28,719 [DEBUG] from application in play-dev-mode-akka.actor.default-dispatcher-8 - Generating patient user.
2019-01-08 06:42:28,726 [DEBUG] from application in play-dev-mode-akka.actor.default-dispatcher-8 - Generating relative user.
2019-01-08 06:42:28,733 [DEBUG] from application in play-dev-mode-akka.actor.default-dispatcher-8 - Generating relative user.
2019-01-08 06:42:28,741 [DEBUG] from application in play-dev-mode-akka.actor.default-dispatcher-8 - Generating relative user.
2019-01-08 06:42:28,758 [INFO] from application in play-dev-mode-akka.actor.default-dispatcher-8 - Test doctors and patients are inserted.
2019-01-08 06:42:28,759 [INFO] from application in play-dev-mode-akka.actor.default-dispatcher-8 - Test nurses are missing. Inserting default values...
2019-01-08 06:42:28,760 [DEBUG] from application in play-dev-mode-akka.actor.default-dispatcher-8 - Generating nurse user.
2019-01-08 06:42:28,767 [DEBUG] from application in play-dev-mode-akka.actor.default-dispatcher-8 - Generating nurse user.
2019-01-08 06:42:28,773 [DEBUG] from application in play-dev-mode-akka.actor.default-dispatcher-8 - Generating nurse user.
2019-01-08 06:42:28,779 [INFO] from application in play-dev-mode-akka.actor.default-dispatcher-8 - Test nurses are inserted.
2019-01-08 06:42:28,780 [INFO] from application in play-dev-mode-akka.actor.default-dispatcher-8 - Admin accounts are missing. Inserting default values...
2019-01-08 06:42:28,788 [INFO] from application in play-dev-mode-akka.actor.default-dispatcher-8 - Admin account is inserted.
2019-01-08 06:42:28,788 [INFO] from application in play-dev-mode-akka.actor.default-dispatcher-8 - DB handler successfully completed!
2019-01-08 06:42:29,280 [WARN] from org.hibernate.validator.messageinterpolation.ParameterMessageInterpolator in play-dev-mode-akka.actor.default-dispatcher-8 - HV000184: ParameterMessageInterpolator has been chosen, EL interpolation will not be supported
2019-01-08 06:42:29,504 [INFO] from play.api.http.EnabledFilters in play-dev-mode-akka.actor.default-dispatcher-8 - Enabled Filters (see <https://www.playframework.com/documentation/latest/Filters>):

    play.filters.headers.SecurityHeadersFilter
    play.filters.hosts.AllowedHostsFilter
    play.filters.cors.CORSFilter

2019-01-08 06:42:29,518 [INFO] from play.api.Play in play-dev-mode-akka.actor.default-dispatcher-8 - Application started (Dev)
2019-01-08 07:23:07,122 [DEBUG] from application in application-akka.actor.default-dispatcher-33 - creating patient...
2019-01-08 07:23:07,122 [DEBUG] from application in application-akka.actor.default-dispatcher-33 - Generating patient user.
2019-01-08 07:23:07,132 [ERROR] from org.jdbcdslog.ResultSetLogger in application-akka.actor.default-dispatcher-33 - java.sql.ResultSet.getObject: 
throws exception: java.lang.ArrayIndexOutOfBoundsException: 2
java.lang.ArrayIndexOutOfBoundsException: 2
	at org.postgresql.util.ByteConverter.int8(ByteConverter.java:27)
	at org.postgresql.jdbc.PgResultSet.getLong(PgResultSet.java:2081)
	at org.postgresql.jdbc.PgResultSet.internalGetObject(PgResultSet.java:185)
	at org.postgresql.jdbc.PgResultSet.getObject(PgResultSet.java:2568)
	at com.zaxxer.hikari.pool.HikariProxyResultSet.getObject(HikariProxyResultSet.java)
	at sun.reflect.GeneratedMethodAccessor20.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.jdbcdslog.ResultSetLoggingHandler.invoke(ResultSetLoggingHandler.java:19)
	at com.sun.proxy.$Proxy40.getObject(Unknown Source)
	at io.ebeaninternal.server.persist.BatchedPstmt.getGeneratedKeys(BatchedPstmt.java:120)
	at io.ebeaninternal.server.persist.BatchedPstmt.executeBatch(BatchedPstmt.java:79)
	at io.ebeaninternal.server.persist.BatchedPstmtHolder.flush(BatchedPstmtHolder.java:93)
	at io.ebeaninternal.server.persist.BatchControl.flushPstmtHolder(BatchControl.java:199)
	at io.ebeaninternal.server.persist.BatchControl.executeNow(BatchControl.java:213)
	at io.ebeaninternal.server.persist.BatchedBeanHolder.executeNow(BatchedBeanHolder.java:88)
	at io.ebeaninternal.server.persist.BatchControl.flush(BatchControl.java:263)
	at io.ebeaninternal.server.persist.BatchControl.flush(BatchControl.java:220)
	at io.ebeaninternal.server.transaction.JdbcTransaction.batchFlush(JdbcTransaction.java:632)
	at io.ebeaninternal.server.transaction.JdbcTransaction.internalBatchFlush(JdbcTransaction.java:750)
	at io.ebeaninternal.server.transaction.JdbcTransaction.flushCommitAndNotify(JdbcTransaction.java:935)
	at io.ebeaninternal.server.transaction.JdbcTransaction.commit(JdbcTransaction.java:987)
	at io.ebeaninternal.server.core.BeanRequest.commitTransIfRequired(BeanRequest.java:61)
	at io.ebeaninternal.server.persist.DefaultPersister.update(DefaultPersister.java:375)
	at io.ebeaninternal.server.persist.DefaultPersister.save(DefaultPersister.java:391)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1578)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1570)
	at io.ebean.Model.save(Model.java:190)
	at controllers.api.DoctorUserController.createRecord(DoctorUserController.java:103)
	at router.Routes$$anonfun$routes$1.$anonfun$applyOrElse$10(Routes.scala:426)
	at play.core.routing.HandlerInvokerFactory$$anon$3.resultCall(HandlerInvoker.scala:134)
	at play.core.routing.HandlerInvokerFactory$$anon$3.resultCall(HandlerInvoker.scala:133)
	at play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$8$$anon$2$$anon$1.invocation(HandlerInvoker.scala:108)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:88)
	at play.http.DefaultActionCreator$1.call(DefaultActionCreator.java:31)
	at play.core.j.JavaAction.$anonfun$apply$8(JavaAction.scala:138)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:655)
	at scala.util.Success.$anonfun$map$1(Try.scala:251)
	at scala.util.Success.map(Try.scala:209)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:289)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:29)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:29)
	at scala.concurrent.impl.CallbackRunnable.run$$$capture(Promise.scala:60)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:56)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:48)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:68)
	at scala.concurrent.impl.Promise$KeptPromise$Kept.onComplete(Promise.scala:368)
	at scala.concurrent.impl.Promise$KeptPromise$Kept.onComplete$(Promise.scala:367)
	at scala.concurrent.impl.Promise$KeptPromise$Successful.onComplete(Promise.scala:375)
	at scala.concurrent.impl.Promise.transform(Promise.scala:29)
	at scala.concurrent.impl.Promise.transform$(Promise.scala:27)
	at scala.concurrent.impl.Promise$KeptPromise$Successful.transform(Promise.scala:375)
	at scala.concurrent.Future.map(Future.scala:289)
	at scala.concurrent.Future.map$(Future.scala:289)
	at scala.concurrent.impl.Promise$KeptPromise$Successful.map(Promise.scala:375)
	at scala.concurrent.Future$.apply(Future.scala:655)
	at play.core.j.JavaAction.apply(JavaAction.scala:138)
	at play.api.mvc.Action.$anonfun$apply$2(Action.scala:96)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:304)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:37)
	at scala.concurrent.impl.CallbackRunnable.run$$$capture(Promise.scala:60)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:55)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:91)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:81)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:91)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:40)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:43)
	at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
2019-01-08 07:23:07,140 [ERROR] from application in application-akka.actor.default-dispatcher-33 - 

! @7aff42j6c - Internal server error, for (POST) [/record] ->
 
play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[CompletionException: javax.persistence.RollbackException: java.lang.ArrayIndexOutOfBoundsException: 2]]
	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:255)
	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:182)
	at play.filters.cors.AbstractCORSPolicy$$anonfun$1.applyOrElse(AbstractCORSPolicy.scala:125)
	at play.filters.cors.AbstractCORSPolicy$$anonfun$1.applyOrElse(AbstractCORSPolicy.scala:123)
	at scala.concurrent.Future.$anonfun$recoverWith$1(Future.scala:414)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:37)
	at scala.concurrent.impl.CallbackRunnable.run$$$capture(Promise.scala:60)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:68)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:284)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:284)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:284)
	at scala.concurrent.impl.Promise$DefaultPromise.link(Promise.scala:337)
	at scala.concurrent.impl.Promise$DefaultPromise.linkRootOf(Promise.scala:322)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:39)
	at scala.concurrent.impl.CallbackRunnable.run$$$capture(Promise.scala:60)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:55)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:91)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:81)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:91)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:40)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:43)
	at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: java.util.concurrent.CompletionException: javax.persistence.RollbackException: java.lang.ArrayIndexOutOfBoundsException: 2
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniApply(CompletableFuture.java:593)
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:577)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1977)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:21)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:18)
	at scala.concurrent.impl.CallbackRunnable.run$$$capture(Promise.scala:60)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:63)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:78)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:81)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:55)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:866)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:106)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:103)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:864)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:68)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:284)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:284)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:284)
	at scala.concurrent.Promise.tryFailure(Promise.scala:108)
	at scala.concurrent.Promise.tryFailure$(Promise.scala:108)
	at scala.concurrent.impl.Promise$DefaultPromise.tryFailure(Promise.scala:183)
	at play.api.mvc.Filter$$anon$1$$anonfun$$nestedInanonfun$apply$4$1.applyOrElse(Filters.scala:81)
	at play.api.mvc.Filter$$anon$1$$anonfun$$nestedInanonfun$apply$4$1.applyOrElse(Filters.scala:75)
	at scala.concurrent.Future.$anonfun$recoverWith$1(Future.scala:414)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:37)
	... 13 common frames omitted
Caused by: javax.persistence.RollbackException: java.lang.ArrayIndexOutOfBoundsException: 2
	at io.ebeaninternal.server.transaction.JdbcTransaction.wrapIfNeeded(JdbcTransaction.java:1007)
	at io.ebeaninternal.server.transaction.JdbcTransaction.commit(JdbcTransaction.java:992)
	at io.ebeaninternal.server.core.BeanRequest.commitTransIfRequired(BeanRequest.java:61)
	at io.ebeaninternal.server.persist.DefaultPersister.update(DefaultPersister.java:375)
	at io.ebeaninternal.server.persist.DefaultPersister.save(DefaultPersister.java:391)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1578)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1570)
	at io.ebean.Model.save(Model.java:190)
	at controllers.api.DoctorUserController.createRecord(DoctorUserController.java:103)
	at router.Routes$$anonfun$routes$1.$anonfun$applyOrElse$10(Routes.scala:426)
	at play.core.routing.HandlerInvokerFactory$$anon$3.resultCall(HandlerInvoker.scala:134)
	at play.core.routing.HandlerInvokerFactory$$anon$3.resultCall(HandlerInvoker.scala:133)
	at play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$8$$anon$2$$anon$1.invocation(HandlerInvoker.scala:108)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:88)
	at play.http.DefaultActionCreator$1.call(DefaultActionCreator.java:31)
	at play.core.j.JavaAction.$anonfun$apply$8(JavaAction.scala:138)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:655)
	at scala.util.Success.$anonfun$map$1(Try.scala:251)
	at scala.util.Success.map(Try.scala:209)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:289)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:29)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:29)
	at scala.concurrent.impl.CallbackRunnable.run$$$capture(Promise.scala:60)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:56)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:48)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:68)
	at scala.concurrent.impl.Promise$KeptPromise$Kept.onComplete(Promise.scala:368)
	at scala.concurrent.impl.Promise$KeptPromise$Kept.onComplete$(Promise.scala:367)
	at scala.concurrent.impl.Promise$KeptPromise$Successful.onComplete(Promise.scala:375)
	at scala.concurrent.impl.Promise.transform(Promise.scala:29)
	at scala.concurrent.impl.Promise.transform$(Promise.scala:27)
	at scala.concurrent.impl.Promise$KeptPromise$Successful.transform(Promise.scala:375)
	at scala.concurrent.Future.map(Future.scala:289)
	at scala.concurrent.Future.map$(Future.scala:289)
	at scala.concurrent.impl.Promise$KeptPromise$Successful.map(Promise.scala:375)
	at scala.concurrent.Future$.apply(Future.scala:655)
	at play.core.j.JavaAction.apply(JavaAction.scala:138)
	at play.api.mvc.Action.$anonfun$apply$2(Action.scala:96)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:304)
	... 14 common frames omitted
Caused by: java.lang.ArrayIndexOutOfBoundsException: 2
	at org.postgresql.util.ByteConverter.int8(ByteConverter.java:27)
	at org.postgresql.jdbc.PgResultSet.getLong(PgResultSet.java:2081)
	at org.postgresql.jdbc.PgResultSet.internalGetObject(PgResultSet.java:185)
	at org.postgresql.jdbc.PgResultSet.getObject(PgResultSet.java:2568)
	at com.zaxxer.hikari.pool.HikariProxyResultSet.getObject(HikariProxyResultSet.java)
	at sun.reflect.GeneratedMethodAccessor20.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.jdbcdslog.ResultSetLoggingHandler.invoke(ResultSetLoggingHandler.java:19)
	at com.sun.proxy.$Proxy40.getObject(Unknown Source)
	at io.ebeaninternal.server.persist.BatchedPstmt.getGeneratedKeys(BatchedPstmt.java:120)
	at io.ebeaninternal.server.persist.BatchedPstmt.executeBatch(BatchedPstmt.java:79)
	at io.ebeaninternal.server.persist.BatchedPstmtHolder.flush(BatchedPstmtHolder.java:93)
	at io.ebeaninternal.server.persist.BatchControl.flushPstmtHolder(BatchControl.java:199)
	at io.ebeaninternal.server.persist.BatchControl.executeNow(BatchControl.java:213)
	at io.ebeaninternal.server.persist.BatchedBeanHolder.executeNow(BatchedBeanHolder.java:88)
	at io.ebeaninternal.server.persist.BatchControl.flush(BatchControl.java:263)
	at io.ebeaninternal.server.persist.BatchControl.flush(BatchControl.java:220)
	at io.ebeaninternal.server.transaction.JdbcTransaction.batchFlush(JdbcTransaction.java:632)
	at io.ebeaninternal.server.transaction.JdbcTransaction.internalBatchFlush(JdbcTransaction.java:750)
	at io.ebeaninternal.server.transaction.JdbcTransaction.flushCommitAndNotify(JdbcTransaction.java:935)
	at io.ebeaninternal.server.transaction.JdbcTransaction.commit(JdbcTransaction.java:987)
	... 53 common frames omitted
2019-01-08 07:23:16,353 [DEBUG] from application in application-akka.actor.default-dispatcher-28 - creating patient...
2019-01-08 07:23:16,353 [DEBUG] from application in application-akka.actor.default-dispatcher-28 - Generating patient user.
2019-01-08 07:23:16,369 [ERROR] from org.jdbcdslog.ResultSetLogger in application-akka.actor.default-dispatcher-28 - java.sql.ResultSet.getObject: 
throws exception: java.lang.ArrayIndexOutOfBoundsException: 2
java.lang.ArrayIndexOutOfBoundsException: 2
	at org.postgresql.util.ByteConverter.int8(ByteConverter.java:27)
	at org.postgresql.jdbc.PgResultSet.getLong(PgResultSet.java:2081)
	at org.postgresql.jdbc.PgResultSet.internalGetObject(PgResultSet.java:185)
	at org.postgresql.jdbc.PgResultSet.getObject(PgResultSet.java:2568)
	at com.zaxxer.hikari.pool.HikariProxyResultSet.getObject(HikariProxyResultSet.java)
	at sun.reflect.GeneratedMethodAccessor20.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.jdbcdslog.ResultSetLoggingHandler.invoke(ResultSetLoggingHandler.java:19)
	at com.sun.proxy.$Proxy40.getObject(Unknown Source)
	at io.ebeaninternal.server.persist.BatchedPstmt.getGeneratedKeys(BatchedPstmt.java:120)
	at io.ebeaninternal.server.persist.BatchedPstmt.executeBatch(BatchedPstmt.java:79)
	at io.ebeaninternal.server.persist.BatchedPstmtHolder.flush(BatchedPstmtHolder.java:93)
	at io.ebeaninternal.server.persist.BatchControl.flushPstmtHolder(BatchControl.java:199)
	at io.ebeaninternal.server.persist.BatchControl.executeNow(BatchControl.java:213)
	at io.ebeaninternal.server.persist.BatchedBeanHolder.executeNow(BatchedBeanHolder.java:88)
	at io.ebeaninternal.server.persist.BatchControl.flush(BatchControl.java:263)
	at io.ebeaninternal.server.persist.BatchControl.flush(BatchControl.java:220)
	at io.ebeaninternal.server.transaction.JdbcTransaction.batchFlush(JdbcTransaction.java:632)
	at io.ebeaninternal.server.transaction.JdbcTransaction.internalBatchFlush(JdbcTransaction.java:750)
	at io.ebeaninternal.server.transaction.JdbcTransaction.flushCommitAndNotify(JdbcTransaction.java:935)
	at io.ebeaninternal.server.transaction.JdbcTransaction.commit(JdbcTransaction.java:987)
	at io.ebeaninternal.server.core.BeanRequest.commitTransIfRequired(BeanRequest.java:61)
	at io.ebeaninternal.server.persist.DefaultPersister.update(DefaultPersister.java:375)
	at io.ebeaninternal.server.persist.DefaultPersister.save(DefaultPersister.java:391)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1578)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1570)
	at io.ebean.Model.save(Model.java:190)
	at controllers.api.DoctorUserController.createRecord(DoctorUserController.java:103)
	at router.Routes$$anonfun$routes$1.$anonfun$applyOrElse$10(Routes.scala:426)
	at play.core.routing.HandlerInvokerFactory$$anon$3.resultCall(HandlerInvoker.scala:134)
	at play.core.routing.HandlerInvokerFactory$$anon$3.resultCall(HandlerInvoker.scala:133)
	at play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$8$$anon$2$$anon$1.invocation(HandlerInvoker.scala:108)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:88)
	at play.http.DefaultActionCreator$1.call(DefaultActionCreator.java:31)
	at play.core.j.JavaAction.$anonfun$apply$8(JavaAction.scala:138)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:655)
	at scala.util.Success.$anonfun$map$1(Try.scala:251)
	at scala.util.Success.map(Try.scala:209)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:289)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:29)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:29)
	at scala.concurrent.impl.CallbackRunnable.run$$$capture(Promise.scala:60)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:56)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:48)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:68)
	at scala.concurrent.impl.Promise$KeptPromise$Kept.onComplete(Promise.scala:368)
	at scala.concurrent.impl.Promise$KeptPromise$Kept.onComplete$(Promise.scala:367)
	at scala.concurrent.impl.Promise$KeptPromise$Successful.onComplete(Promise.scala:375)
	at scala.concurrent.impl.Promise.transform(Promise.scala:29)
	at scala.concurrent.impl.Promise.transform$(Promise.scala:27)
	at scala.concurrent.impl.Promise$KeptPromise$Successful.transform(Promise.scala:375)
	at scala.concurrent.Future.map(Future.scala:289)
	at scala.concurrent.Future.map$(Future.scala:289)
	at scala.concurrent.impl.Promise$KeptPromise$Successful.map(Promise.scala:375)
	at scala.concurrent.Future$.apply(Future.scala:655)
	at play.core.j.JavaAction.apply(JavaAction.scala:138)
	at play.api.mvc.Action.$anonfun$apply$2(Action.scala:96)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:304)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:37)
	at scala.concurrent.impl.CallbackRunnable.run$$$capture(Promise.scala:60)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:55)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:91)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:81)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:91)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:40)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:43)
	at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
2019-01-08 07:23:16,370 [ERROR] from application in application-akka.actor.default-dispatcher-28 - 

! @7aff436nd - Internal server error, for (POST) [/record] ->
 
play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[CompletionException: javax.persistence.RollbackException: java.lang.ArrayIndexOutOfBoundsException: 2]]
	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:255)
	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:182)
	at play.filters.cors.AbstractCORSPolicy$$anonfun$1.applyOrElse(AbstractCORSPolicy.scala:125)
	at play.filters.cors.AbstractCORSPolicy$$anonfun$1.applyOrElse(AbstractCORSPolicy.scala:123)
	at scala.concurrent.Future.$anonfun$recoverWith$1(Future.scala:414)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:37)
	at scala.concurrent.impl.CallbackRunnable.run$$$capture(Promise.scala:60)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:68)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:284)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:284)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:284)
	at scala.concurrent.impl.Promise$DefaultPromise.link(Promise.scala:337)
	at scala.concurrent.impl.Promise$DefaultPromise.linkRootOf(Promise.scala:322)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:39)
	at scala.concurrent.impl.CallbackRunnable.run$$$capture(Promise.scala:60)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:55)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:91)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:81)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:91)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:40)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:43)
	at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: java.util.concurrent.CompletionException: javax.persistence.RollbackException: java.lang.ArrayIndexOutOfBoundsException: 2
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniApply(CompletableFuture.java:593)
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:577)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1977)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:21)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:18)
	at scala.concurrent.impl.CallbackRunnable.run$$$capture(Promise.scala:60)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:63)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:78)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:81)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:55)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:866)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:106)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:103)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:864)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:68)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:284)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:284)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:284)
	at scala.concurrent.Promise.tryFailure(Promise.scala:108)
	at scala.concurrent.Promise.tryFailure$(Promise.scala:108)
	at scala.concurrent.impl.Promise$DefaultPromise.tryFailure(Promise.scala:183)
	at play.api.mvc.Filter$$anon$1$$anonfun$$nestedInanonfun$apply$4$1.applyOrElse(Filters.scala:81)
	at play.api.mvc.Filter$$anon$1$$anonfun$$nestedInanonfun$apply$4$1.applyOrElse(Filters.scala:75)
	at scala.concurrent.Future.$anonfun$recoverWith$1(Future.scala:414)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:37)
	... 13 common frames omitted
Caused by: javax.persistence.RollbackException: java.lang.ArrayIndexOutOfBoundsException: 2
	at io.ebeaninternal.server.transaction.JdbcTransaction.wrapIfNeeded(JdbcTransaction.java:1007)
	at io.ebeaninternal.server.transaction.JdbcTransaction.commit(JdbcTransaction.java:992)
	at io.ebeaninternal.server.core.BeanRequest.commitTransIfRequired(BeanRequest.java:61)
	at io.ebeaninternal.server.persist.DefaultPersister.update(DefaultPersister.java:375)
	at io.ebeaninternal.server.persist.DefaultPersister.save(DefaultPersister.java:391)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1578)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1570)
	at io.ebean.Model.save(Model.java:190)
	at controllers.api.DoctorUserController.createRecord(DoctorUserController.java:103)
	at router.Routes$$anonfun$routes$1.$anonfun$applyOrElse$10(Routes.scala:426)
	at play.core.routing.HandlerInvokerFactory$$anon$3.resultCall(HandlerInvoker.scala:134)
	at play.core.routing.HandlerInvokerFactory$$anon$3.resultCall(HandlerInvoker.scala:133)
	at play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$8$$anon$2$$anon$1.invocation(HandlerInvoker.scala:108)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:88)
	at play.http.DefaultActionCreator$1.call(DefaultActionCreator.java:31)
	at play.core.j.JavaAction.$anonfun$apply$8(JavaAction.scala:138)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:655)
	at scala.util.Success.$anonfun$map$1(Try.scala:251)
	at scala.util.Success.map(Try.scala:209)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:289)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:29)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:29)
	at scala.concurrent.impl.CallbackRunnable.run$$$capture(Promise.scala:60)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:56)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:48)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:68)
	at scala.concurrent.impl.Promise$KeptPromise$Kept.onComplete(Promise.scala:368)
	at scala.concurrent.impl.Promise$KeptPromise$Kept.onComplete$(Promise.scala:367)
	at scala.concurrent.impl.Promise$KeptPromise$Successful.onComplete(Promise.scala:375)
	at scala.concurrent.impl.Promise.transform(Promise.scala:29)
	at scala.concurrent.impl.Promise.transform$(Promise.scala:27)
	at scala.concurrent.impl.Promise$KeptPromise$Successful.transform(Promise.scala:375)
	at scala.concurrent.Future.map(Future.scala:289)
	at scala.concurrent.Future.map$(Future.scala:289)
	at scala.concurrent.impl.Promise$KeptPromise$Successful.map(Promise.scala:375)
	at scala.concurrent.Future$.apply(Future.scala:655)
	at play.core.j.JavaAction.apply(JavaAction.scala:138)
	at play.api.mvc.Action.$anonfun$apply$2(Action.scala:96)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:304)
	... 14 common frames omitted
Caused by: java.lang.ArrayIndexOutOfBoundsException: 2
	at org.postgresql.util.ByteConverter.int8(ByteConverter.java:27)
	at org.postgresql.jdbc.PgResultSet.getLong(PgResultSet.java:2081)
	at org.postgresql.jdbc.PgResultSet.internalGetObject(PgResultSet.java:185)
	at org.postgresql.jdbc.PgResultSet.getObject(PgResultSet.java:2568)
	at com.zaxxer.hikari.pool.HikariProxyResultSet.getObject(HikariProxyResultSet.java)
	at sun.reflect.GeneratedMethodAccessor20.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.jdbcdslog.ResultSetLoggingHandler.invoke(ResultSetLoggingHandler.java:19)
	at com.sun.proxy.$Proxy40.getObject(Unknown Source)
	at io.ebeaninternal.server.persist.BatchedPstmt.getGeneratedKeys(BatchedPstmt.java:120)
	at io.ebeaninternal.server.persist.BatchedPstmt.executeBatch(BatchedPstmt.java:79)
	at io.ebeaninternal.server.persist.BatchedPstmtHolder.flush(BatchedPstmtHolder.java:93)
	at io.ebeaninternal.server.persist.BatchControl.flushPstmtHolder(BatchControl.java:199)
	at io.ebeaninternal.server.persist.BatchControl.executeNow(BatchControl.java:213)
	at io.ebeaninternal.server.persist.BatchedBeanHolder.executeNow(BatchedBeanHolder.java:88)
	at io.ebeaninternal.server.persist.BatchControl.flush(BatchControl.java:263)
	at io.ebeaninternal.server.persist.BatchControl.flush(BatchControl.java:220)
	at io.ebeaninternal.server.transaction.JdbcTransaction.batchFlush(JdbcTransaction.java:632)
	at io.ebeaninternal.server.transaction.JdbcTransaction.internalBatchFlush(JdbcTransaction.java:750)
	at io.ebeaninternal.server.transaction.JdbcTransaction.flushCommitAndNotify(JdbcTransaction.java:935)
	at io.ebeaninternal.server.transaction.JdbcTransaction.commit(JdbcTransaction.java:987)
	... 53 common frames omitted
2019-01-08 07:23:24,916 [DEBUG] from application in application-akka.actor.default-dispatcher-33 - creating patient...
2019-01-08 07:23:24,916 [DEBUG] from application in application-akka.actor.default-dispatcher-33 - Generating patient user.
2019-01-08 07:23:24,936 [ERROR] from org.jdbcdslog.ResultSetLogger in application-akka.actor.default-dispatcher-33 - java.sql.ResultSet.getObject: 
throws exception: java.lang.ArrayIndexOutOfBoundsException: 2
java.lang.ArrayIndexOutOfBoundsException: 2
	at org.postgresql.util.ByteConverter.int8(ByteConverter.java:27)
	at org.postgresql.jdbc.PgResultSet.getLong(PgResultSet.java:2081)
	at org.postgresql.jdbc.PgResultSet.internalGetObject(PgResultSet.java:185)
	at org.postgresql.jdbc.PgResultSet.getObject(PgResultSet.java:2568)
	at com.zaxxer.hikari.pool.HikariProxyResultSet.getObject(HikariProxyResultSet.java)
	at sun.reflect.GeneratedMethodAccessor20.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.jdbcdslog.ResultSetLoggingHandler.invoke(ResultSetLoggingHandler.java:19)
	at com.sun.proxy.$Proxy40.getObject(Unknown Source)
	at io.ebeaninternal.server.persist.BatchedPstmt.getGeneratedKeys(BatchedPstmt.java:120)
	at io.ebeaninternal.server.persist.BatchedPstmt.executeBatch(BatchedPstmt.java:79)
	at io.ebeaninternal.server.persist.BatchedPstmtHolder.flush(BatchedPstmtHolder.java:93)
	at io.ebeaninternal.server.persist.BatchControl.flushPstmtHolder(BatchControl.java:199)
	at io.ebeaninternal.server.persist.BatchControl.executeNow(BatchControl.java:213)
	at io.ebeaninternal.server.persist.BatchedBeanHolder.executeNow(BatchedBeanHolder.java:88)
	at io.ebeaninternal.server.persist.BatchControl.flush(BatchControl.java:263)
	at io.ebeaninternal.server.persist.BatchControl.flush(BatchControl.java:220)
	at io.ebeaninternal.server.transaction.JdbcTransaction.batchFlush(JdbcTransaction.java:632)
	at io.ebeaninternal.server.transaction.JdbcTransaction.internalBatchFlush(JdbcTransaction.java:750)
	at io.ebeaninternal.server.transaction.JdbcTransaction.flushCommitAndNotify(JdbcTransaction.java:935)
	at io.ebeaninternal.server.transaction.JdbcTransaction.commit(JdbcTransaction.java:987)
	at io.ebeaninternal.server.core.BeanRequest.commitTransIfRequired(BeanRequest.java:61)
	at io.ebeaninternal.server.persist.DefaultPersister.update(DefaultPersister.java:375)
	at io.ebeaninternal.server.persist.DefaultPersister.save(DefaultPersister.java:391)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1578)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1570)
	at io.ebean.Model.save(Model.java:190)
	at controllers.api.DoctorUserController.createRecord(DoctorUserController.java:103)
	at router.Routes$$anonfun$routes$1.$anonfun$applyOrElse$10(Routes.scala:426)
	at play.core.routing.HandlerInvokerFactory$$anon$3.resultCall(HandlerInvoker.scala:134)
	at play.core.routing.HandlerInvokerFactory$$anon$3.resultCall(HandlerInvoker.scala:133)
	at play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$8$$anon$2$$anon$1.invocation(HandlerInvoker.scala:108)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:88)
	at play.http.DefaultActionCreator$1.call(DefaultActionCreator.java:31)
	at play.core.j.JavaAction.$anonfun$apply$8(JavaAction.scala:138)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:655)
	at scala.util.Success.$anonfun$map$1(Try.scala:251)
	at scala.util.Success.map(Try.scala:209)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:289)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:29)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:29)
	at scala.concurrent.impl.CallbackRunnable.run$$$capture(Promise.scala:60)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:56)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:48)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:68)
	at scala.concurrent.impl.Promise$KeptPromise$Kept.onComplete(Promise.scala:368)
	at scala.concurrent.impl.Promise$KeptPromise$Kept.onComplete$(Promise.scala:367)
	at scala.concurrent.impl.Promise$KeptPromise$Successful.onComplete(Promise.scala:375)
	at scala.concurrent.impl.Promise.transform(Promise.scala:29)
	at scala.concurrent.impl.Promise.transform$(Promise.scala:27)
	at scala.concurrent.impl.Promise$KeptPromise$Successful.transform(Promise.scala:375)
	at scala.concurrent.Future.map(Future.scala:289)
	at scala.concurrent.Future.map$(Future.scala:289)
	at scala.concurrent.impl.Promise$KeptPromise$Successful.map(Promise.scala:375)
	at scala.concurrent.Future$.apply(Future.scala:655)
	at play.core.j.JavaAction.apply(JavaAction.scala:138)
	at play.api.mvc.Action.$anonfun$apply$2(Action.scala:96)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:304)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:37)
	at scala.concurrent.impl.CallbackRunnable.run$$$capture(Promise.scala:60)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:55)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:91)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:81)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:91)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:40)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:43)
	at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
2019-01-08 07:23:24,938 [ERROR] from application in application-akka.actor.default-dispatcher-33 - 

! @7aff43jf0 - Internal server error, for (POST) [/record] ->
 
play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[CompletionException: javax.persistence.RollbackException: java.lang.ArrayIndexOutOfBoundsException: 2]]
	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:255)
	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:182)
	at play.filters.cors.AbstractCORSPolicy$$anonfun$1.applyOrElse(AbstractCORSPolicy.scala:125)
	at play.filters.cors.AbstractCORSPolicy$$anonfun$1.applyOrElse(AbstractCORSPolicy.scala:123)
	at scala.concurrent.Future.$anonfun$recoverWith$1(Future.scala:414)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:37)
	at scala.concurrent.impl.CallbackRunnable.run$$$capture(Promise.scala:60)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:68)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:284)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:284)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:284)
	at scala.concurrent.impl.Promise$DefaultPromise.link(Promise.scala:337)
	at scala.concurrent.impl.Promise$DefaultPromise.linkRootOf(Promise.scala:322)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:39)
	at scala.concurrent.impl.CallbackRunnable.run$$$capture(Promise.scala:60)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:55)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:91)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:81)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:91)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:40)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:43)
	at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: java.util.concurrent.CompletionException: javax.persistence.RollbackException: java.lang.ArrayIndexOutOfBoundsException: 2
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniApply(CompletableFuture.java:593)
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:577)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1977)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:21)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:18)
	at scala.concurrent.impl.CallbackRunnable.run$$$capture(Promise.scala:60)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:63)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:78)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:81)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:55)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:866)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:106)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:103)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:864)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:68)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:284)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:284)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:284)
	at scala.concurrent.Promise.tryFailure(Promise.scala:108)
	at scala.concurrent.Promise.tryFailure$(Promise.scala:108)
	at scala.concurrent.impl.Promise$DefaultPromise.tryFailure(Promise.scala:183)
	at play.api.mvc.Filter$$anon$1$$anonfun$$nestedInanonfun$apply$4$1.applyOrElse(Filters.scala:81)
	at play.api.mvc.Filter$$anon$1$$anonfun$$nestedInanonfun$apply$4$1.applyOrElse(Filters.scala:75)
	at scala.concurrent.Future.$anonfun$recoverWith$1(Future.scala:414)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:37)
	... 13 common frames omitted
Caused by: javax.persistence.RollbackException: java.lang.ArrayIndexOutOfBoundsException: 2
	at io.ebeaninternal.server.transaction.JdbcTransaction.wrapIfNeeded(JdbcTransaction.java:1007)
	at io.ebeaninternal.server.transaction.JdbcTransaction.commit(JdbcTransaction.java:992)
	at io.ebeaninternal.server.core.BeanRequest.commitTransIfRequired(BeanRequest.java:61)
	at io.ebeaninternal.server.persist.DefaultPersister.update(DefaultPersister.java:375)
	at io.ebeaninternal.server.persist.DefaultPersister.save(DefaultPersister.java:391)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1578)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1570)
	at io.ebean.Model.save(Model.java:190)
	at controllers.api.DoctorUserController.createRecord(DoctorUserController.java:103)
	at router.Routes$$anonfun$routes$1.$anonfun$applyOrElse$10(Routes.scala:426)
	at play.core.routing.HandlerInvokerFactory$$anon$3.resultCall(HandlerInvoker.scala:134)
	at play.core.routing.HandlerInvokerFactory$$anon$3.resultCall(HandlerInvoker.scala:133)
	at play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$8$$anon$2$$anon$1.invocation(HandlerInvoker.scala:108)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:88)
	at play.http.DefaultActionCreator$1.call(DefaultActionCreator.java:31)
	at play.core.j.JavaAction.$anonfun$apply$8(JavaAction.scala:138)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:655)
	at scala.util.Success.$anonfun$map$1(Try.scala:251)
	at scala.util.Success.map(Try.scala:209)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:289)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:29)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:29)
	at scala.concurrent.impl.CallbackRunnable.run$$$capture(Promise.scala:60)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:56)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:48)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:68)
	at scala.concurrent.impl.Promise$KeptPromise$Kept.onComplete(Promise.scala:368)
	at scala.concurrent.impl.Promise$KeptPromise$Kept.onComplete$(Promise.scala:367)
	at scala.concurrent.impl.Promise$KeptPromise$Successful.onComplete(Promise.scala:375)
	at scala.concurrent.impl.Promise.transform(Promise.scala:29)
	at scala.concurrent.impl.Promise.transform$(Promise.scala:27)
	at scala.concurrent.impl.Promise$KeptPromise$Successful.transform(Promise.scala:375)
	at scala.concurrent.Future.map(Future.scala:289)
	at scala.concurrent.Future.map$(Future.scala:289)
	at scala.concurrent.impl.Promise$KeptPromise$Successful.map(Promise.scala:375)
	at scala.concurrent.Future$.apply(Future.scala:655)
	at play.core.j.JavaAction.apply(JavaAction.scala:138)
	at play.api.mvc.Action.$anonfun$apply$2(Action.scala:96)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:304)
	... 14 common frames omitted
Caused by: java.lang.ArrayIndexOutOfBoundsException: 2
	at org.postgresql.util.ByteConverter.int8(ByteConverter.java:27)
	at org.postgresql.jdbc.PgResultSet.getLong(PgResultSet.java:2081)
	at org.postgresql.jdbc.PgResultSet.internalGetObject(PgResultSet.java:185)
	at org.postgresql.jdbc.PgResultSet.getObject(PgResultSet.java:2568)
	at com.zaxxer.hikari.pool.HikariProxyResultSet.getObject(HikariProxyResultSet.java)
	at sun.reflect.GeneratedMethodAccessor20.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.jdbcdslog.ResultSetLoggingHandler.invoke(ResultSetLoggingHandler.java:19)
	at com.sun.proxy.$Proxy40.getObject(Unknown Source)
	at io.ebeaninternal.server.persist.BatchedPstmt.getGeneratedKeys(BatchedPstmt.java:120)
	at io.ebeaninternal.server.persist.BatchedPstmt.executeBatch(BatchedPstmt.java:79)
	at io.ebeaninternal.server.persist.BatchedPstmtHolder.flush(BatchedPstmtHolder.java:93)
	at io.ebeaninternal.server.persist.BatchControl.flushPstmtHolder(BatchControl.java:199)
	at io.ebeaninternal.server.persist.BatchControl.executeNow(BatchControl.java:213)
	at io.ebeaninternal.server.persist.BatchedBeanHolder.executeNow(BatchedBeanHolder.java:88)
	at io.ebeaninternal.server.persist.BatchControl.flush(BatchControl.java:263)
	at io.ebeaninternal.server.persist.BatchControl.flush(BatchControl.java:220)
	at io.ebeaninternal.server.transaction.JdbcTransaction.batchFlush(JdbcTransaction.java:632)
	at io.ebeaninternal.server.transaction.JdbcTransaction.internalBatchFlush(JdbcTransaction.java:750)
	at io.ebeaninternal.server.transaction.JdbcTransaction.flushCommitAndNotify(JdbcTransaction.java:935)
	at io.ebeaninternal.server.transaction.JdbcTransaction.commit(JdbcTransaction.java:987)
	... 53 common frames omitted
2019-01-08 07:23:39,681 [DEBUG] from application in application-akka.actor.default-dispatcher-34 - creating patient...
2019-01-08 07:23:39,682 [DEBUG] from application in application-akka.actor.default-dispatcher-34 - Generating patient user.
2019-01-08 07:23:39,695 [ERROR] from org.jdbcdslog.ResultSetLogger in application-akka.actor.default-dispatcher-34 - java.sql.ResultSet.getObject: 
throws exception: java.lang.ArrayIndexOutOfBoundsException: 2
java.lang.ArrayIndexOutOfBoundsException: 2
	at org.postgresql.util.ByteConverter.int8(ByteConverter.java:27)
	at org.postgresql.jdbc.PgResultSet.getLong(PgResultSet.java:2081)
	at org.postgresql.jdbc.PgResultSet.internalGetObject(PgResultSet.java:185)
	at org.postgresql.jdbc.PgResultSet.getObject(PgResultSet.java:2568)
	at com.zaxxer.hikari.pool.HikariProxyResultSet.getObject(HikariProxyResultSet.java)
	at sun.reflect.GeneratedMethodAccessor20.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.jdbcdslog.ResultSetLoggingHandler.invoke(ResultSetLoggingHandler.java:19)
	at com.sun.proxy.$Proxy40.getObject(Unknown Source)
	at io.ebeaninternal.server.persist.BatchedPstmt.getGeneratedKeys(BatchedPstmt.java:120)
	at io.ebeaninternal.server.persist.BatchedPstmt.executeBatch(BatchedPstmt.java:79)
	at io.ebeaninternal.server.persist.BatchedPstmtHolder.flush(BatchedPstmtHolder.java:93)
	at io.ebeaninternal.server.persist.BatchControl.flushPstmtHolder(BatchControl.java:199)
	at io.ebeaninternal.server.persist.BatchControl.executeNow(BatchControl.java:213)
	at io.ebeaninternal.server.persist.BatchedBeanHolder.executeNow(BatchedBeanHolder.java:88)
	at io.ebeaninternal.server.persist.BatchControl.flush(BatchControl.java:263)
	at io.ebeaninternal.server.persist.BatchControl.flush(BatchControl.java:220)
	at io.ebeaninternal.server.transaction.JdbcTransaction.batchFlush(JdbcTransaction.java:632)
	at io.ebeaninternal.server.transaction.JdbcTransaction.internalBatchFlush(JdbcTransaction.java:750)
	at io.ebeaninternal.server.transaction.JdbcTransaction.flushCommitAndNotify(JdbcTransaction.java:935)
	at io.ebeaninternal.server.transaction.JdbcTransaction.commit(JdbcTransaction.java:987)
	at io.ebeaninternal.server.core.BeanRequest.commitTransIfRequired(BeanRequest.java:61)
	at io.ebeaninternal.server.persist.DefaultPersister.update(DefaultPersister.java:375)
	at io.ebeaninternal.server.persist.DefaultPersister.save(DefaultPersister.java:391)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1578)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1570)
	at io.ebean.Model.save(Model.java:190)
	at controllers.api.DoctorUserController.createRecord(DoctorUserController.java:103)
	at router.Routes$$anonfun$routes$1.$anonfun$applyOrElse$10(Routes.scala:426)
	at play.core.routing.HandlerInvokerFactory$$anon$3.resultCall(HandlerInvoker.scala:134)
	at play.core.routing.HandlerInvokerFactory$$anon$3.resultCall(HandlerInvoker.scala:133)
	at play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$8$$anon$2$$anon$1.invocation(HandlerInvoker.scala:108)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:88)
	at play.http.DefaultActionCreator$1.call(DefaultActionCreator.java:31)
	at play.core.j.JavaAction.$anonfun$apply$8(JavaAction.scala:138)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:655)
	at scala.util.Success.$anonfun$map$1(Try.scala:251)
	at scala.util.Success.map(Try.scala:209)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:289)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:29)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:29)
	at scala.concurrent.impl.CallbackRunnable.run$$$capture(Promise.scala:60)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:56)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:48)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:68)
	at scala.concurrent.impl.Promise$KeptPromise$Kept.onComplete(Promise.scala:368)
	at scala.concurrent.impl.Promise$KeptPromise$Kept.onComplete$(Promise.scala:367)
	at scala.concurrent.impl.Promise$KeptPromise$Successful.onComplete(Promise.scala:375)
	at scala.concurrent.impl.Promise.transform(Promise.scala:29)
	at scala.concurrent.impl.Promise.transform$(Promise.scala:27)
	at scala.concurrent.impl.Promise$KeptPromise$Successful.transform(Promise.scala:375)
	at scala.concurrent.Future.map(Future.scala:289)
	at scala.concurrent.Future.map$(Future.scala:289)
	at scala.concurrent.impl.Promise$KeptPromise$Successful.map(Promise.scala:375)
	at scala.concurrent.Future$.apply(Future.scala:655)
	at play.core.j.JavaAction.apply(JavaAction.scala:138)
	at play.api.mvc.Action.$anonfun$apply$2(Action.scala:96)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:304)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:37)
	at scala.concurrent.impl.CallbackRunnable.run$$$capture(Promise.scala:60)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:55)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:91)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:81)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:91)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:40)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:43)
	at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
2019-01-08 07:23:39,697 [ERROR] from application in application-akka.actor.default-dispatcher-34 - 

! @7aff44fah - Internal server error, for (POST) [/record] ->
 
play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[CompletionException: javax.persistence.RollbackException: java.lang.ArrayIndexOutOfBoundsException: 2]]
	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:255)
	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:182)
	at play.filters.cors.AbstractCORSPolicy$$anonfun$1.applyOrElse(AbstractCORSPolicy.scala:125)
	at play.filters.cors.AbstractCORSPolicy$$anonfun$1.applyOrElse(AbstractCORSPolicy.scala:123)
	at scala.concurrent.Future.$anonfun$recoverWith$1(Future.scala:414)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:37)
	at scala.concurrent.impl.CallbackRunnable.run$$$capture(Promise.scala:60)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:68)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:284)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:284)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:284)
	at scala.concurrent.impl.Promise$DefaultPromise.link(Promise.scala:337)
	at scala.concurrent.impl.Promise$DefaultPromise.linkRootOf(Promise.scala:322)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:39)
	at scala.concurrent.impl.CallbackRunnable.run$$$capture(Promise.scala:60)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:55)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:91)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:81)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:91)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:40)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:43)
	at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: java.util.concurrent.CompletionException: javax.persistence.RollbackException: java.lang.ArrayIndexOutOfBoundsException: 2
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniApply(CompletableFuture.java:593)
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:577)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1977)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:21)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:18)
	at scala.concurrent.impl.CallbackRunnable.run$$$capture(Promise.scala:60)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:63)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:78)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:81)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:55)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:866)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:106)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:103)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:864)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:68)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:284)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:284)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:284)
	at scala.concurrent.Promise.tryFailure(Promise.scala:108)
	at scala.concurrent.Promise.tryFailure$(Promise.scala:108)
	at scala.concurrent.impl.Promise$DefaultPromise.tryFailure(Promise.scala:183)
	at play.api.mvc.Filter$$anon$1$$anonfun$$nestedInanonfun$apply$4$1.applyOrElse(Filters.scala:81)
	at play.api.mvc.Filter$$anon$1$$anonfun$$nestedInanonfun$apply$4$1.applyOrElse(Filters.scala:75)
	at scala.concurrent.Future.$anonfun$recoverWith$1(Future.scala:414)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:37)
	... 13 common frames omitted
Caused by: javax.persistence.RollbackException: java.lang.ArrayIndexOutOfBoundsException: 2
	at io.ebeaninternal.server.transaction.JdbcTransaction.wrapIfNeeded(JdbcTransaction.java:1007)
	at io.ebeaninternal.server.transaction.JdbcTransaction.commit(JdbcTransaction.java:992)
	at io.ebeaninternal.server.core.BeanRequest.commitTransIfRequired(BeanRequest.java:61)
	at io.ebeaninternal.server.persist.DefaultPersister.update(DefaultPersister.java:375)
	at io.ebeaninternal.server.persist.DefaultPersister.save(DefaultPersister.java:391)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1578)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1570)
	at io.ebean.Model.save(Model.java:190)
	at controllers.api.DoctorUserController.createRecord(DoctorUserController.java:103)
	at router.Routes$$anonfun$routes$1.$anonfun$applyOrElse$10(Routes.scala:426)
	at play.core.routing.HandlerInvokerFactory$$anon$3.resultCall(HandlerInvoker.scala:134)
	at play.core.routing.HandlerInvokerFactory$$anon$3.resultCall(HandlerInvoker.scala:133)
	at play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$8$$anon$2$$anon$1.invocation(HandlerInvoker.scala:108)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:88)
	at play.http.DefaultActionCreator$1.call(DefaultActionCreator.java:31)
	at play.core.j.JavaAction.$anonfun$apply$8(JavaAction.scala:138)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:655)
	at scala.util.Success.$anonfun$map$1(Try.scala:251)
	at scala.util.Success.map(Try.scala:209)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:289)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:29)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:29)
	at scala.concurrent.impl.CallbackRunnable.run$$$capture(Promise.scala:60)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:56)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:48)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:68)
	at scala.concurrent.impl.Promise$KeptPromise$Kept.onComplete(Promise.scala:368)
	at scala.concurrent.impl.Promise$KeptPromise$Kept.onComplete$(Promise.scala:367)
	at scala.concurrent.impl.Promise$KeptPromise$Successful.onComplete(Promise.scala:375)
	at scala.concurrent.impl.Promise.transform(Promise.scala:29)
	at scala.concurrent.impl.Promise.transform$(Promise.scala:27)
	at scala.concurrent.impl.Promise$KeptPromise$Successful.transform(Promise.scala:375)
	at scala.concurrent.Future.map(Future.scala:289)
	at scala.concurrent.Future.map$(Future.scala:289)
	at scala.concurrent.impl.Promise$KeptPromise$Successful.map(Promise.scala:375)
	at scala.concurrent.Future$.apply(Future.scala:655)
	at play.core.j.JavaAction.apply(JavaAction.scala:138)
	at play.api.mvc.Action.$anonfun$apply$2(Action.scala:96)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:304)
	... 14 common frames omitted
Caused by: java.lang.ArrayIndexOutOfBoundsException: 2
	at org.postgresql.util.ByteConverter.int8(ByteConverter.java:27)
	at org.postgresql.jdbc.PgResultSet.getLong(PgResultSet.java:2081)
	at org.postgresql.jdbc.PgResultSet.internalGetObject(PgResultSet.java:185)
	at org.postgresql.jdbc.PgResultSet.getObject(PgResultSet.java:2568)
	at com.zaxxer.hikari.pool.HikariProxyResultSet.getObject(HikariProxyResultSet.java)
	at sun.reflect.GeneratedMethodAccessor20.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.jdbcdslog.ResultSetLoggingHandler.invoke(ResultSetLoggingHandler.java:19)
	at com.sun.proxy.$Proxy40.getObject(Unknown Source)
	at io.ebeaninternal.server.persist.BatchedPstmt.getGeneratedKeys(BatchedPstmt.java:120)
	at io.ebeaninternal.server.persist.BatchedPstmt.executeBatch(BatchedPstmt.java:79)
	at io.ebeaninternal.server.persist.BatchedPstmtHolder.flush(BatchedPstmtHolder.java:93)
	at io.ebeaninternal.server.persist.BatchControl.flushPstmtHolder(BatchControl.java:199)
	at io.ebeaninternal.server.persist.BatchControl.executeNow(BatchControl.java:213)
	at io.ebeaninternal.server.persist.BatchedBeanHolder.executeNow(BatchedBeanHolder.java:88)
	at io.ebeaninternal.server.persist.BatchControl.flush(BatchControl.java:263)
	at io.ebeaninternal.server.persist.BatchControl.flush(BatchControl.java:220)
	at io.ebeaninternal.server.transaction.JdbcTransaction.batchFlush(JdbcTransaction.java:632)
	at io.ebeaninternal.server.transaction.JdbcTransaction.internalBatchFlush(JdbcTransaction.java:750)
	at io.ebeaninternal.server.transaction.JdbcTransaction.flushCommitAndNotify(JdbcTransaction.java:935)
	at io.ebeaninternal.server.transaction.JdbcTransaction.commit(JdbcTransaction.java:987)
	... 53 common frames omitted
2019-01-08 07:26:28,099 [INFO] from application in application-akka.actor.default-dispatcher-36 - Shutting down connection pool.
2019-01-08 07:26:28,282 [INFO] from application in play-dev-mode-akka.actor.default-dispatcher-16 - Creating Pool for datasource 'default'
2019-01-08 07:26:28,290 [INFO] from play.api.db.DefaultDBApi in play-dev-mode-akka.actor.default-dispatcher-16 - Database [default] connected at jdbc:postgresql://localhost/ceng472
2019-01-08 07:26:28,357 [INFO] from application in play-dev-mode-akka.actor.default-dispatcher-16 - DB handler is started...
2019-01-08 07:26:28,361 [INFO] from application in play-dev-mode-akka.actor.default-dispatcher-16 - DB handler successfully completed!
2019-01-08 07:26:28,455 [WARN] from org.hibernate.validator.messageinterpolation.ParameterMessageInterpolator in play-dev-mode-akka.actor.default-dispatcher-16 - HV000184: ParameterMessageInterpolator has been chosen, EL interpolation will not be supported
2019-01-08 07:26:28,635 [INFO] from play.api.http.EnabledFilters in play-dev-mode-akka.actor.default-dispatcher-16 - Enabled Filters (see <https://www.playframework.com/documentation/latest/Filters>):

    play.filters.headers.SecurityHeadersFilter
    play.filters.hosts.AllowedHostsFilter
    play.filters.cors.CORSFilter

2019-01-08 07:26:28,636 [INFO] from play.api.Play in play-dev-mode-akka.actor.default-dispatcher-16 - Application started (Dev)
2019-01-08 07:26:28,700 [DEBUG] from application in application-akka.actor.default-dispatcher-5 - creating patient...
2019-01-08 07:26:28,700 [DEBUG] from application in application-akka.actor.default-dispatcher-5 - Generating patient user.
2019-01-08 07:36:08,452 [DEBUG] from application in application-akka.actor.default-dispatcher-27 - creating relative...
2019-01-08 07:36:08,452 [DEBUG] from application in application-akka.actor.default-dispatcher-27 - Generating relative user.
